src/client/app/components/DashboardComponent.tsx

```diff
@@ -11,60 +11,132 @@ import MultiCompareChartContainer from '../containers/MultiCompareChartContainer
 import SpinnerComponent from './SpinnerComponent';
 import { ChartTypes } from '../types/redux/graph';
 
+import * as moment from 'moment';
+import { TimeInterval } from '../../../common/TimeInterval';
+
+import Button from 'reactstrap/lib/Button';
+import { FormEvent } from 'react';
+import { FormattedMessage } from 'react-intl';
+import { Dispatch, Thunk, ActionType } from '../types/redux/actions';
+
+import * as Plotly from 'plotly.js';
```

Comment:
This is a little picky but you use lowercase plotly for the import in index.d.ts but uppercase here.